modules/tinymce/src/core/test/ts/browser/keyboard/PageUpDownKeysTest.ts

```diff
@@ -0,0 +1,87 @@
+import { Keys } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyContentActions, TinyHooks, TinySelections } from '@ephox/mcagar';
+import { PlatformDetection } from '@ephox/sand';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.core.keyboard.PageUpDownKeyTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    add_unload_trigger: false,
+    base_url: '/project/tinymce/js/tinymce',
+    indent: false
+  }, [ Theme ], true);
+  const platform = PlatformDetection.detect();
+
+  context('Page Up', () => {
+    it('caret is escape outside the node', () => {
```

Comment:
Maybe this should be `Caret should be placed before the inline element if it's at the start of the inline element` or something?

- [ ] Go though the titles of the tests seems like they could be made more clear what they are supposed to be doing

Reply:
updated